lib/pubid/iso/harmonized_stage_code.rb

```diff
@@ -0,0 +1,50 @@
+module Pubid::Iso
+  class HarmonizedStageCode
+    attr_accessor :stage, :substage
+
+    STAGES = { PWI: "00.00",
```

Comment:
Let's not have these "abbreviated stages" in this class. These codes only applies to ISO documents, they are not inherent to the HSC (harmonized stage codes). Let's move this outside of this class but keep in ISO.

Reply:
@ronaldtse I moved it to separate class "Stage". Could you check the implementation?